test/parse/yaml.go

```diff
@@ -23,6 +23,17 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+// MustParseV1alpha1StepAction takes YAML and parses it into a *v1alpha1.StepAction
+func MustParseV1alpha1StepAction(t *testing.T, yaml string) *v1alpha1.StepAction {
```

Comment:
Good catch. I think it will be used in the next PR. I split up the large one into smaller ones so that's how it must have slipped in.

I removed it from here.

Thanks!